frontend/src/Utils.js

```diff
@@ -17,24 +17,62 @@ export const fetchSongData = async (songName, setCurrSong) => {
   }
 };
 
-export const fetchSonsgByName = async (songName, setSongs, count = 6) => {
-  try {
-    const response = await axios.get(`https://spring-music-player-3hyj.vercel.app/search?song=${encodeURIComponent(songName)}`);
-    const jsonData = response.data;
-    const songs = jsonData.slice(0, count);
-    setSongs(songs);
-  } catch (error) {
-    console.error("There was a problem with the fetch operation:", error);
-  }
+// fetch song by its name
+export const fetchSongsByName = async (songName, setSongs, count = 6) => {
+  fetch(
+    `https://jio-savaan-private.vercel.app/api/search/songs?query=${encodeURIComponent(songName)}`
```

Comment:
don't change backend api now.

Reply:
here only function name that is 
fetchSonsgByName -> fetchSongsByName I changed

Reply:
https://spring-music-player-3hyj.vercel.app utilize this url